sdk/webpubsub/azure-messaging-webpubsub-client/src/main/java/com/azure/messaging/webpubsub/client/implementation/models/WebPubSubMessageAck.java

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.webpubsub.client.implementation.models;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+
+public abstract class WebPubSubMessageAck extends WebPubSubMessage {
```

Comment:
What's the relationship between `WebPubSubMessageAck` and `AckMessage`?

Reply:
AckMessage is the wire message (but not visible to user).

WebPubSubMessageAck is the response for user-facing API.

They are similar.

Reply:
WebPubSubMessageAck is just a convenience class, it is not on wire.

Impl just want to have a single class WebPubSubMessageAck to call `getAckId()` and `setAckId()` (for 4 of its subclasses)